src/main/webapp/index.html

```diff
@@ -26,20 +27,58 @@ <h3>a recommendation system for students, by students!</h3>
         </ul>
       </nav>
     </div>    
-    <h3>Recommendation Map</h3>
+    <section class="map-section">
+      <div class="content">
+        <div class="info-paragraph">
+          <h3>Want to add a recommendation?</h3>
+          <p>No problem! Click on the map to add a new marker and then
+            you can add your recommendation. Be sure to let everyone
+            know your favourite places!</p>
+        </div>
+        <div class="info-paragraph">
+          <h3>Want to view a recommendation?</h3>
+          <p>To see recommendations, use the buttons above to select a 
+            category and then explore the markers on the map! Have a 
+            good look around and see if anything catches your eye.</p>
+        </div>
+      </div>
     <div class="map-category-section">
       <button class="map-category-button">Restaurants</button>
       <button class="map-category-button">Places to Visit</button>
       <button class="map-category-button">Bars and Clubs</button>
       <button class="map-category-button">Study Places</button>
     </div>
+    </section>
     <!-- The Recommendation Map -->
     <div id="map"></div>
-    
-    <div class="map-info">
-      <h2>Want to add a recommendation?</h2>
-      <p>No problem! Click on the map to add a new marker and to create
-        your recommendation.</p>
+    <!-- Popup for Adding Recommendation -->
+    <div class="popup" id="popup-add-recs">
```

Comment:
I think u can remove class="popup" and in the css just put #popup-add-recs instead of .popup

Reply:
Yes, I can do that. I chose to have it this way though because I thought we would reuse some aspects of the formatting in other popups, but I'll make it just #popup-add-recs for now, since we currently don't have any other popups.